.idea/runConfigurations/Artemis__Server_.xml

```diff
@@ -3,7 +3,6 @@
     <module name="Artemis.main" />
     <option name="SPRING_BOOT_MAIN_CLASS" value="de.tum.in.www1.artemis.ArtemisApp" />
     <option name="ACTIVE_PROFILES" value="dev,bamboo,bitbucket,jira,artemis,scheduling,local" />
-    <option name="VM_PARAMETERS" value="--illegal-access=warn" />
```

Comment:
These changes are good, but I'm not sure whether they really belong to this PR.

Reply:
Then please simply ignore it. Is it really such a big problem for you that you spend time complaining about it? 😅